plugins/direct_events_plugin/db/services.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+defined('MOODLE_INTERNAL') || die();
+
+$services = array(
+      'directeventhandler' => array(                                                // the name of the web service
```

Comment:
It'd be nice to use a more human friendly string here in stead of `directeventhandler` as it shows up in the UI. Maybe `FE Events (Direct) service` is more appropriate?

Reply:
Nudging this comment from the first pass.